### PR TITLE
110 banner ad and link to emoji detail screen

### PR DESCRIPTION
**Pull request by taleeikenes**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -50,4 +51,5 @@ easily add support for multiple languages if needed -->
     <string name="no_favorites">You don\'t have any favorites yet. Navigate to an emoji to start
         creating your first emoji sequence</string>
     <string name="games_icon_description">Games</string>
+>>>>>>> dev
```
Comment (OleAndreasER) [inline app/src/main/res/values/strings.xml]: ">>>>>>> dev" is from the merge. It shouldn't be here.

File: app/src/main/kotlin/com/platform/openemoji/screens/EmojiDetailScreen.kt [hunk 2]
```diff
@@ -50,11 +59,34 @@ fun EmojiDetailScreen(
...
+        HtmlCompat.fromHtml(
+            emoji.description,
+            HtmlCompat.FROM_HTML_MODE_COMPACT,
+        ).toString().trim() + "\n"
+    val seeMore = stringResource(R.string.see_more_details)
+    val annotatedString =
+        buildAnnotatedString {
+            withStyle(style = MaterialTheme.typography.titleMedium.toSpanStyle()) {
+                append(description)
+            }
```
Comment (OleAndreasER) [inline app/src/main/kotlin/com/platform/openemoji/screens/EmojiDetailScreen.kt]: Description needs material theme color. It's currently the same color as background on my phone.

File: app/src/main/kotlin/com/platform/openemoji/screens/EmojiDetailScreen.kt [hunk 3]
```diff
@@ -50,11 +59,34 @@ fun EmojiDetailScreen(
...
         }
     }
+    val scrollState = rememberScrollState()
+    val description =
+        HtmlCompat.fromHtml(
+            emoji.description,
+            HtmlCompat.FROM_HTML_MODE_COMPACT,
+        ).toString().trim() + "\n"
+    val seeMore = stringResource(R.string.see_more_details)
+    val annotatedString =
```
Comment (OleAndreasER) [inline app/src/main/kotlin/com/platform/openemoji/screens/EmojiDetailScreen.kt]: "annotatedString" -> "styledEmojiDescription" or add a comment so it's easier for the reader to figure out what this is.

**Review by OleAndreasER — CHANGES_REQUESTED** [review]
Good. It works. Just a few small things.

Don't you think the banner ad should be on all screens? You could put it in Scaffold topBar.

File: app/src/main/kotlin/com/platform/openemoji/screens/EmojiDetailScreen.kt (hunk 2 above)
Comment (taleeikenes) [reply app/src/main/kotlin/com/platform/openemoji/screens/EmojiDetailScreen.kt]: If I add color - it disappears for me, but I added this:
`withStyle(
                style =
                    MaterialTheme.typography.titleMedium.toSpanStyle().copy(
                        color = MaterialTheme.colorScheme.onSurface,
                    ),
            )`
Does it work for you then?

**Pull request merged**